components/follower/Follower.tsx

```diff
@@ -0,0 +1,80 @@
+import { useRecoilValue } from 'recoil';
+import styled from 'styled-components';
+import { followerListState } from '../../recoil/followerList';
+import { useState } from 'react';
+
+interface FollowButtonProps {
+  isFollowing: boolean;
+}
+
+const Follower = () => {
+  const followerList = useRecoilValue(followerListState);
+  const followerNameList = followerList.map((it) => it.login);
```

Comment:
요렇게 구조분해 할당으로도 사용할 수 있써요..!  지금은 it에서 login만 가져와서 괜찮지만 나중에 많은 요소들을 가져와야 할 때 구조분해 할당을 해주면 조금 더 가독성이 좋아지는거 같더라구요..!
```suggestion
const followerNameList = followerList.map(({login}) => login);
```